### ISSUE TITLE
Continuous Integration should test both qiskit and pyquil integrations of the library

### ISSUE THREAD

**Issue by willzeng** [unitaryfund/mitiq#4]

**Comment by rmlarose** [comment]
We had issues getting quilc and the qvm to run on Travis for NISQAI -- is this the current issue? If so,

https://github.com/rmlarose/nisqai-dev/blob/dev/.travis.yml

may be of assistance. I forget the actual fix because we kept fudging things until it finally worked.

**Comment by willzeng** [comment]
Yep I remember emailing with you about that. That's the way I planned to get it to work, but for some reason it wasn't finding the mapped port from the docker images. Since I couldn't find an interactive more for github actions build system I'm still not sure how to best debug. Let me know if you find one.

An alternative fix could be just to switch to Travis or semaphore...
